tests/test_buffer.py

```diff
@@ -450,6 +450,15 @@ def test_SimpleArray_from_ndarray_content(self):
         sarr.ndarray.fill(100)
         self.assertTrue((ndarr == 100).all())
 
+    def test_SimpleArray_from_ndarray_slice(self):
+        ndarr = np.arange(1000, dtype='float64').reshape((10, 10, 10))
+        parr = ndarr[1:7, 2:6, 3:9]
+        sarr = modmesh.SimpleArrayFloat64(array=ndarr[1:7, 2:6, 3:9])
+        for i in range(6):
+            for j in range(4):
+                for k in range(6):
+                    self.assertEqual(parr[i, j, k], sarr[i, j, k])
```

Comment:
The unittest of constructing SimpleArray from slicing ndarray. When the dimension is less than 4, the constructor should work regularly.